src/cryptoadvance/specter/rpc.py

```diff
@@ -264,8 +264,28 @@ def multi(self, calls: list, **kwargs):
         url = self.url
         if "wallet" in kwargs:
             url = url+"/wallet/{}".format(kwargs["wallet"])
-        r = requests.post(
-            url, data=json.dumps(payload), headers=headers, timeout=timeout)
+        r = None
+        if '.onion' in url:
+            try:
+                requests_session = requests.Session()
+                requests_session.proxies = {}
```

Comment:
Is `requests_session.proxies = {}` needed?

`Session` object already [initialize](https://github.com/psf/requests/blob/967a05bfffcb68f97296eda197b062221c2ebc0d/requests/sessions.py#L373) the attribute `proxies` to {}

Reply:
Right, that's redundant, removed. Thanks for the review!